modules/repository/check.go

```diff
@@ -72,13 +72,18 @@ func GitGcRepos(ctx context.Context) error {
 			if err := repo.GetOwner(); err != nil {
 				return err
 			}
+			log.Trace("Running git gc on %v", repo)
```

Comment:
What logging config do I need to see these? I have `LEVEL = Trace` and tried both file/stdout logging. Also, I'd suggest raiding these to `Info`.